src/environments/environment.prod.ts

```diff
@@ -1,3 +1,7 @@
 export const environment = {
-  production: true
+  production: true,
```

Comment:
How about using a predicate-like method naming: 

```suggestion
  isProduction: true,
```